exp-player/addon/randomizers/pref-phys.js

```diff
@@ -10,105 +12,1336 @@ function shuffleArray(array) {
+        ['cup', 'whiteball', 'lotion', 'spray', 'apple', 'orangeball'],
+        ['orangeball', 'cup', 'lotion', 'spray', 'apple', 'whiteball'],
+        ['apple', 'whiteball', 'lotion', 'cup', 'spray', 'orangeball'],
+        ['whiteball', 'apple', 'orangeball', 'spray', 'cup', 'lotion'],
+        ['cup', 'whiteball', 'orangeball', 'spray', 'apple', 'lotion'],
+        ['orangeball', 'lotion', 'apple', 'whiteball', 'cup', 'spray'],
+        ['whiteball', 'orangeball', 'lotion', 'spray', 'apple', 'cup'],
+        ['whiteball', 'cup', 'lotion', 'spray', 'apple', 'orangeball'],
+        ['apple', 'cup', 'orangeball', 'whiteball', 'lotion', 'spray'],
+        ['orangeball', 'lotion', 'apple', 'cup', 'spray', 'whiteball'],
+        ['apple', 'lotion', 'cup', 'spray', 'whiteball', 'orangeball'],
+        ['cup', 'orangeball', 'lotion', 'spray', 'apple', 'whiteball'],
+        ['whiteball', 'orangeball', 'cup', 'apple', 'spray', 'lotion'],
+        ['cup', 'lotion', 'whiteball', 'spray', 'apple', 'orangeball'],
+        ['cup', 'orangeball', 'lotion', 'spray', 'whiteball', 'apple'],
+        ['lotion', 'cup', 'orangeball', 'spray', 'whiteball', 'apple'],
+        ['lotion', 'orangeball', 'cup', 'whiteball', 'apple', 'spray'],
+        ['apple', 'whiteball', 'cup', 'lotion', 'spray', 'orangeball'],
+        ['cup', 'lotion', 'apple', 'spray', 'whiteball', 'orangeball'],
+        ['apple', 'lotion', 'orangeball', 'spray', 'whiteball', 'cup'],
+        ['whiteball', 'cup', 'orangeball', 'apple', 'spray', 'lotion'],
+        ['whiteball', 'apple', 'orangeball', 'spray', 'lotion', 'cup'],
+        ['whiteball', 'cup', 'lotion', 'apple', 'spray', 'orangeball'],
+        ['apple', 'orangeball', 'cup', 'spray', 'whiteball', 'lotion'],
+        ['whiteball', 'apple', 'orangeball', 'cup', 'lotion', 'spray'],
+        ['apple', 'whiteball', 'orangeball', 'spray', 'cup', 'lotion'],
+        ['cup', 'apple', 'orangeball', 'lotion', 'whiteball', 'spray'],
+        ['cup', 'orangeball', 'apple', 'lotion', 'whiteball', 'spray'],
+        ['whiteball', 'cup', 'orangeball', 'lotion', 'spray', 'apple'],
+        ['orangeball', 'whiteball', 'apple', 'cup', 'spray', 'lotion'],
+        ['whiteball', 'orangeball', 'apple', 'lotion', 'spray', 'cup'],
+        ['cup', 'orangeball', 'whiteball', 'apple', 'spray', 'lotion'],
+        ['apple', 'whiteball', 'lotion', 'spray', 'cup', 'orangeball'],
+        ['orangeball', 'cup', 'whiteball', 'spray', 'apple', 'lotion'],
+        ['orangeball', 'apple', 'cup', 'lotion', 'whiteball', 'spray'],
+        ['cup', 'apple', 'lotion', 'spray', 'whiteball', 'orangeball'],
+        ['orangeball', 'apple', 'whiteball', 'spray', 'cup', 'lotion'],
+        ['cup', 'orangeball', 'whiteball', 'lotion', 'spray', 'apple'],
+        ['orangeball', 'whiteball', 'apple', 'cup', 'lotion', 'spray'],
+        ['whiteball', 'lotion', 'orangeball', 'apple', 'spray', 'cup'],
+        ['whiteball', 'lotion', 'orangeball', 'apple', 'cup', 'spray']
```

Comment:
Okay, let's just go ahead with the hard-coded lists. I do appreciate the caution, though.

We'd actually have needed to be extremely careful not to change anything about this randomizer regardless; part of my concern about procedurally generating the fixed pseudo-random lists on the fly is that it wouldn't be as glaringly obvious that something had subtly changed (even though changing the random seed, or changing whether the state updated between different lists, or changing the order in which the lists were generated, etc. would violate the expected behavior just as badly as introducing a new 720-line list).